pennylane/noise/conditionals.py

```diff
@@ -757,7 +758,8 @@ def _partial(wires=None, **partial_kwargs):
                     return tuple(operation(**op_args, wires=wire) for wire in op_wires)
 
         if is_mappable and operation.wires is not None:
-            return operation.map_wires(dict(zip(operation.wires, op_args.pop("wires"))))
+            wire_map = dict(zip(operation.wires, op_args.pop("wires")))
+            return map_wires(operation, wire_map, queue=QueuingManager.recording())
```

Comment:
I'm bit concerned why this conditional branch would require the queue's attention since from its context and all other branches this is not quite obvious to me